$NetBSD: patch-data_meson.build,v 1.1 2022/03/11 07:45:02 nia Exp $

Remove unused positional arguments. Previously these were ignored,
now they cause build failures.

--- data/meson.build.orig	2020-10-02 12:27:26.648589800 +0000
+++ data/meson.build
@@ -30,7 +30,6 @@ desktop_conf = configuration_data()
 desktop_conf.set('icon', application_id)
 
 desktop = i18n.merge_file(
-  'desktop',
   input: configure_file(
     input: files('org.gnome.Nautilus.desktop.in.in'),
     output: 'org.gnome.Nautilus.desktop.in',
@@ -44,7 +43,6 @@ desktop = i18n.merge_file(
 )
 
 desktop_autorun_software = i18n.merge_file(
-  'desktop-autorun-software',
   input: 'nautilus-autorun-software.desktop.in',
   output: 'nautilus-autorun-software.desktop',
   install: true,
@@ -58,7 +56,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('appid', application_id)
 
 appdata = i18n.merge_file(
-  'appdata',
   input: configure_file(
     input: files('org.gnome.Nautilus.appdata.xml.in.in'),
     output: 'org.gnome.Nautilus.appdata.xml.in',
